### DISCUSSION TITLE
Return to last scrollbar position in apps list after clicking "< GO BACK" in app details

### DISCUSSION THREAD

**Discussion by rwperrott** [AppOutlet/AppOutlet#278] [Ideas]
I was curious about this app after seeing it on a GitHub Explore email, but I discovered a serious UX issue.

The annoyingly absent icons, and details errors were bearable annoyances, but I judge always returned to the start of the apps list, after clicking on "< GO BACK" in app details, as a serious UX issue, because for apps lists which can't all fit on the screen, the poor user may have to keep re-scrolling forwards to re-find their last position, when looking at multiple app details; been there! Yes, a user could search for an app, but that is missing the point, and a search may return a long list too.

**Comment by MessiasLima** [comment]
Hmm... what I think you are reporting is a list does not retain its last highlighted app ... sounds like a great first issue to dive into for an inspiring dev.
